### ISSUE TITLE
ConceptDriftStream Logging

### ISSUE THREAD

**Issue by foxriver76** [scikit-multiflow/scikit-multiflow#47]
I'm playing around with the Concept Drift Stream Generator with the following code:

```python
from skmultiflow.evaluation.evaluate_prequential import EvaluatePrequential
from skmultiflow.trees.hoeffding_adaptive_tree import HAT
from skmultiflow.data.concept_drift_stream import ConceptDriftStream
from skmultiflow.data import AGRAWALGenerator

"""1. Create stream"""
stream = ConceptDriftStream(stream_option=AGRAWALGenerator(random_state=112), 
                            drift_stream_option=AGRAWALGenerator(random_state=112, classification_function=2),
                            random_state=None,
                            alpha_option=0.0, # angle of change grade 0 - 90
                            position_option=250000,
                            width_option=10000)

stream.prepare_for_use()

"""2. Create classifier"""
clf = HAT(split_criterion='info_gain')

"""3. Setup evaluator"""
evaluator = EvaluatePrequential(show_plot=True,
                                pretrain_size=1,
                                max_samples=1000000,
                                metrics=['performance', 'kappa_t', 'kappa_m', 'kappa'],
                                output_file=None)

"""4. Run evaluator"""
evaluator.evaluate(stream=stream, model=clf, model_names=['HAT']) 
```

When running the code it is very slow due to the enormous amount of logging, like:

```
root - INFO - -50000.0%
root - INFO - -50050.0%
root - INFO - -50100.0%
root - INFO - -50150.0%
root - INFO - -50200.0%
root - INFO - -50250.0%
root - INFO - -50300.0%
root - INFO - -50350.0%
root - INFO - -50400.0%
root - INFO - -50450.0%
root - INFO - -50500.0%
root - INFO - -50550.0%
root - INFO - -50600.0%
root - INFO - -50650.0%
root - INFO - -50700.0%
root - INFO - -50750.0%
root - INFO - -50800.0%
root - INFO - -50850.0%
root - INFO - -50900.0%
root - INFO - -50950.0%
root - INFO - -51000.0%
root - INFO - -51050.0%
root - INFO - -51100.0%
root - INFO - -51150.0%
root - INFO - -51200.0%
root - INFO - -51250.0%
root - INFO - -51300.0%
root - INFO - -51350.0%
root - INFO - -51400.0%
root - INFO - -51450.0%
root - INFO - -51500.0%
root - INFO - -51550.0%
root - INFO - -51600.0%
root - INFO - -51650.0%
root - INFO - -51700.0%
root - INFO - -51750.0%
```

Also I don't know what that means for me.

**Comment by jacobmontiel** [comment]
Hi @foxriver76 

I have pushed a fix for this issue (8af6fcad2572cf679985b500e8330b4a558d113f). The number of remaining samples was not calculated correctly. This triggered the strange behavior in the evaluator.

You will see that the logging is now to normal.
```
root - INFO - Prequential Evaluation
root - INFO - Evaluating 1 target(s).
root - INFO - Pre-training on 1 samples.
root - INFO - Evaluating...
root - INFO - 5.0%
root - INFO - 10.0%
root - INFO - 15.0%
root - INFO - 20.0%
root - INFO - 25.0%
root - INFO - 30.0%
root - INFO - 35.0%
root - INFO - 40.0%
root - INFO - 45.0%
root - INFO - 50.0%
root - INFO - 55.0%
root - INFO - 60.0%
root - INFO - 65.0%
root - INFO - 70.0%
root - INFO - 75.0%
root - INFO - 80.0%
root - INFO - 85.0%
root - INFO - 90.0%
root - INFO - 95.0%
root - INFO - Evaluation time: 7.327 s
root - INFO - Total samples: 10000
root - INFO - Global performance:
root - INFO - HAT - Accuracy     : 0.924
root - INFO - HAT - Kappa        : 0.827
root - INFO - HAT - Kappa T      : 0.827
root - INFO - HAT - Kappa M      : 0.774
```
Thanks for reporting this.

**Comment by foxriver76** [comment]
Thanks for the quick fix.
